recipes-browser/cog/cog_0.17.1.bb

```diff
@@ -0,0 +1,18 @@
+require cog.inc
+require cog-meson.inc
+require conf/include/devupstream.inc
```

Comment:
you need to remove `require conf/include/devupstream.inc` from `recipes-browser/cog/cog_0.16.1.bb:require conf/include/devupstream.inc` also.